configuration/jsr223.md

```diff
@@ -158,58 +158,83 @@ The Default preset is preloaded, so it does not require importing.
 
 | Variable | Description |
 |---------|-------------|
-| `State` | `org.eclipse.smarthome.core.types.State` |
-| `Command` | `org.eclipse.smarthome.core.types.State` |
 | `DateTime` | `org.joda.time.DateTime` (if Jodatime is available) |
 | `LocalTime` | `org.joda.time.LocalTime` (if Jodatime is available) |
+| | |
+| `State` | `org.eclipse.smarthome.core.types.State` |
+| `Command` | `org.eclipse.smarthome.core.types.State` |
+| | |
 | `StringUtils` | `org.apache.commons.lang.StringUtils` |
 | `URLEncoder` | `java.net.URLEncoder` |
 | `FileUtils` | `org.apache.commons.io.FileUtils` |
 | `FilenameUtils` | `org.apache.commons.io.FilenameUtils` |
 | `File` | `java.io.File` |
-| `UnDefType` | `org.eclipse.smarthome.core.library.types.UnDefType` |
-| `NULL` | `UnDefType` enum item |
-| `UNDEF` | `UnDefType` enum item |
+| | |
```

Comment:
Did you have a look at those empty table rows in the preview?
I think they look terrible.

Reply:
I added them because I was comparing the doc directly with the code, which has them logically separated like that. I got used to seeing them like that, so I left them. It will also be easier to compare to the code next time. But if you think they are ugly, I can get rid of them.

Reply:
Maybe separate tables are a better way of dividing content groups.
For me it is displayed ugly, because the tables have striped backgrounds and we cant control which background color will be presented in such an empty row.

So we also cant guarantee a equivalent displayed separator with such an empty line.

>I'm also curious... these changes are accurate for OH 2.5. Is there a way to get them into those docs too, or will they only show for "latest"?

A merge into master here will show this just for the "latest" docs.
If i remember right, we have talked about that with @ghys already some time ago.
We should be able to cherry pick the merged commit into the 2.5.0 tag and it wwill then also be shown in the 2.5.0 versioned docs.

Reply:
> We should be able to cherry pick the merged commit into the 2.5.0 tag and it wwill then also be shown in the 2.5.0 versioned docs.

Unfortunately, it's a little more complicated than that... The docs for a specific version are built manually and integrated into the public folder of the website as static content, because we can't afford rebuilding them every time: if they ever change they have to be built by hand (there's a environment variable you can set to specify the tag/branch, see https://github.com/openhab/website/issues/72) and copied over again with a PR. This is not ideal, I know. Not sure yet how to improve this, maybe with separate subdomains for each version maybe we could map them to branches in Netlify.

Reply:
I think we have to overthink this in general now.
With OH3 in the back we might have to deal with several docs imrpovements that will be necessarily included for 2.5

Reply:
@openhab-5iver 

Did you have a chance to think about thoe view options?
We could merge and cherry pick this one after improving it.

Reply:
I've cleaned up the blank lines and a couple other things. Sorry for the delay!

Reply:
BTW, there will be changes needed to the packages names for the OH3 version.

Reply:
>BTW, there will be changes needed to the packages names for the OH3 version.

So we should cherry pick the changes from here and push an update afterwards directly.